scripts/core/common.yml

```diff
@@ -410,6 +410,9 @@ etors:
     - name: DEVICE_IP_VERSION_EXT
       desc: $x_device_ip_version_ext_t
       version: "1.5"
+    - name: SUB_ALLOCATIONS_EXT_PROPERTIES
+      desc: $x_sub_allocations_ext_properties_t
+      version: "1.5"
```

Comment:
Since this is currently an experimental extension I think we only need the "EXP version below and this "EXT" version should be removed.